internal/runners/packages/packages.go

```diff
@@ -42,7 +47,11 @@ type configurable interface {
 
 const latestVersion = "latest"
 
-func executePackageOperation(pj *project.Project, cfg configurable, out output.Outputer, authentication *authentication.Auth, prompt prompt.Prompter, name, version string, operation model.Operation, ns model.Namespace) error {
+func executePackageOperation(pj *project.Project, cfg configurable, out output.Outputer, authentication *authentication.Auth, prompt prompt.Prompter, packageName, packageVersion, languageName string, operation model.Operation, ns model.Namespace) error {
+	if pj == nil {
+		return installInitial(cfg, out, authentication, prompt, packageName, packageVersion, languageName, operation, ns)
+	}
```

Comment:
It doesn't make sense that we would just skip all the normal operations in favor of this special behavior. I think what we need to do at this point is create the project file, and then the rest of the logic can run as it normally would.